guides/guide.txt

```diff
@@ -0,0 +1,18 @@
+TOPIC
```

Comment:
While fun, I think this clutters the guides list more than anything.

Reply:
I thought about removing it and making it even more of an easter egg - would that work?

Reply:
I think that'd work :)

Reply:
Also definitely open to more whimsical text here - I mostly wanted to avoid getting too close to https://forge.puppet.com/modules/binford2k/easter_egg